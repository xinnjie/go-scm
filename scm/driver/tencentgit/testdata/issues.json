[
  {
    "labels": [
      "test_label"
    ],
    "priority": null,
    "id": 32589,
    "project_id": 179129,
    "title": "test issue",
    "state": "opened",
    "resolve_state": null,
    "iid": 1,
    "description": "issue for test",
    "confidential": false,
    "created_at": "2021-08-26T05:15:43+0000",
    "updated_at": "2021-08-26T05:15:43+0000",
    "author": {
      "id": 176405,
      "username": "xinnjie",
      "web_url": "https://git.code.tencent.com/u/xinnjie",
      "name": "xinnjie",
      "state": "active",
      "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
    },
    "assignees": [
      {
        "id": 176405,
        "username": "xinnjie",
        "web_url": "https://git.code.tencent.com/u/xinnjie",
        "name": "xinnjie",
        "state": "active",
        "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
      }
    ],
    "assignee": {
      "id": 176405,
      "username": "xinnjie",
      "web_url": "https://git.code.tencent.com/u/xinnjie",
      "name": "xinnjie",
      "state": "active",
      "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
    }
  }
]