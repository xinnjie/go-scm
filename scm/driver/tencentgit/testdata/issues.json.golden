[
  {
    "Number": 1,
    "Title": "test issue",
    "Body": "issue for test",
    "State": "open",
    "Link": "https://git.code.tencent.com/xinnjie/testme/issues/1",
    "Labels": [
      "test_label"
    ],
    "Closed": false,
    "Locked": false,
    "Author": {
      "Login": "xinnjie",
      "Name": "xinnjie",
      "Email": "",
      "Avatar": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
    },
    "Assignees": [
      {
        "ID": 176405,
        "Login": "xinnjie",
        "Name": "xinnjie",
        "Link": "https://git.code.tencent.com/u/xinnjie",
        "Avatar":  "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
      }
    ],
    "Created": "2021-08-26T05:15:43.000Z",
    "Updated": "2021-08-26T05:15:43.000Z"
  }
]
